packages/core/src/components/axes/axis.ts

```diff
@@ -395,10 +395,10 @@ export class Axis extends Component {
 					break;
 				case AxisPositions.RIGHT:
 					axisTitleRef
-						.attr("transform", "rotate(90)")
-						.attr("y", -width)
-						.attr("x", scale.range()[0] / 2)
-						.attr("dy", "1em")
+						.attr("transform", "rotate(-90)")
+						.attr("y", width)
+						.attr("x", -(scale.range()[0] / 2))
+						.attr("dy", "-0.5em")
```

Comment:
does this break any other charts?

Reply:
it affects but doesnt break the other axis charts, but im now wondering if this is ideal:
![image](https://user-images.githubusercontent.com/14351335/104510288-f3596e80-55b8-11eb-8611-84d705a8533a.png)

The conversation to have the vertical titles oriented in same direction (and that being to align with the left axis title orientation) didnt go as far as to cover if they should still be facing the other direction when there is just a right axis. I don't see why this (above) is illogical but from the globalization standpoint, maybe we would want all labels facing the other direction in some cases? Would we want to provide a configuration to override the title orientation on each axis ?? 

this way a user would get the chart above **but they can modify it to get this:**
![image](https://user-images.githubusercontent.com/14351335/104509497-c48ec880-55b7-11eb-8223-6dff3e49c3fb.png)

but again, not sure about the criticality of the * direction in which the titles face** so long as its consistent

Reply:
cc @jeanservaas